spec/vehicle_spec.rb

```diff
@@ -15,6 +16,22 @@
       expect(@cruz.model).to eq('Cruz')
       expect(@cruz.engine).to eq(:ice)
       expect(@cruz.registration_date).to eq(nil)
+
+      expect(@bolt).to be_an_instance_of(Vehicle)
+      expect(@bolt.vin).to eq('987654321abcdefgh')
+      expect(@bolt.year).to eq(2019)
+      expect(@bolt.make).to eq('Chevrolet')
+      expect(@bolt.model).to eq('Bolt')
+      expect(@bolt.engine).to eq(:ev)
+      expect(@bolt.registration_date).to eq(nil)
+
+      expect(@camaro).to be_an_instance_of(Vehicle)
+      expect(@camaro.vin).to eq('1a2b3c4d5e6f')
+      expect(@camaro.year).to eq(1969)
+      expect(@camaro.make).to eq('Chevrolet')
+      expect(@camaro.model).to eq('Camaro')
+      expect(@camaro.engine).to eq(:ice)
+      expect(@camaro.registration_date).to eq(nil)
```

Comment:
I would also test in your `vehicile_spec` your `new_registration_date` method since that's a method that you defined in the Vehicle class.